cli/azd/pkg/azdo/service_connection.go

```diff
@@ -103,10 +103,6 @@ func CreateServiceConnection(
 
 	// if a service connection exists, skip creating a new Service connection. But update the current connection only
 	if foundServiceConnection != nil {
-		console.Message(
-			ctx,
-			output.WithWarningFormat("Service Connection %s already exists. Updating endpoint", ServiceConnectionName),
-		)
```

Comment:
I'm assuming we're intentionally removing this warning now?

Reply:
yes, it's not part of the UX design